Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -1215,6 +1338,42 @@ void HGCalGeomParameters::loadSpecParsTrapezoid(const HGCalParameters& php) {
   for (unsigned int k = 0; k < php.layerCenter_.size(); ++k)
     edm::LogVerbatim("HGCalGeom") << "[" << k << "] " << php.layerCenter_[k];
 #endif
+
+  // tile parameters from Katja's file
+  if (php.waferMaskMode_ == katjaFile) {
+    for (unsigned int k = 0; k < tileIndx_.size(); ++k) {
+      php.tileInfoMap_[tileIndx_[k]] =
+          HGCalParameters::tileInfo(tileType_[k], tileSiPM_[k], tileHEX1_[k], tileHEX2_[k], tileHEX3_[k], tileHEX4_[k]);
+#ifdef EDM_ML_DEBUG
+      edm::LogVerbatim("HGCalGeom") << "Tile[" << k << ":" << tileIndx_[k] << "] "
+                                    << " Type " << tileType_[k] << " SiPM " << tileSiPM_[k] << " HEX " << tileHEX1_[k]
+                                    << ":" << tileHEX2_[k] << ":" << tileHEX3_[k] << ":" << tileHEX4_[k];
+#endif
+    }
+    tileIndx_.clear();
+    tileType_.clear();
+    tileSiPM_.clear();
+    tileHEX1_.clear();
+    tileHEX2_.clear();
+    tileHEX3_.clear();
+    tileHEX4_.clear();
+    for (unsigned int k = 0; k < tileRMin_.size(); ++k) {
+      php.tileRingR_.emplace_back(std::make_pair(tileRMin_[k], tileRMax_[k]));
```

Comment:
don't need to use make_pair with emplace_back

Reply:
Done